README.md

```diff
@@ -39,78 +39,95 @@
 | Experimental | Bash · C · C++ · Clojure · Dart · Dockerfile · Elixir · HTML · Julia · Jsonnet · Lisp · Lua · OCaml · R · Scheme · Solidity · Swift · YAML · XML · Generic (ERB, Jinja, etc.) |
 
-### Getting started 🚀
 
-1. [From the CLI](#option-1-getting-started-from-the-cli)
-2. [From the Semgrep Cloud Platform](#option-2-getting-started-from-the-semgrep-cloud-platform-recommended)
+Semgrep Supply Chain supports 8 languages across over 15 package managers. 
 
-For beginners, we recommend starting with the [Semgrep Cloud Platform](#option-2-getting-started-from-the-semgrep-cloud-platform-recommended) because it provides a visual interface, a demo project, result triaging and exploration workflows, and makes setup in CI/CD fast. Scans are still local and code isn't uploaded. Alternatively, you can also start with the CLI without logging in and navigate the terminal output to run one-off searches.
+| Category     | Languages                                                                                                                                                                     |
+| -------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| GA             | Go (Go modules, `go mod`) · Javascript/Typescript (npm, Yarn, Yarn 2, Yarn 3, pnpm) · Python (pip, pip-tool, Pipenv, Poetry) · Ruby (RubyGems) · Java (Gradle, Maven)       |
+| Beta           | C# (NuGet)                                                                                                                                                                  |
+| Lock file-only | Rust (Cargo) · PHP (Composer) |
 
-### Option 1: Getting started from the CLI
+For more information, visit our [supported languages](https://semgrep.dev/docs/supported-languages/) page. 
 
-1.  Install Semgrep CLI
+### Getting started 🚀
 
-```
-# For macOS
-$ brew install semgrep
+1. [From the Semgrep Cloud Platform](#option-1-getting-started-from-the-semgrep-cloud-platform-recommended)
+2. [From the CLI](#option-2-getting-started-from-the-cli)
 
-# For Ubuntu/WSL/Linux/macOS
-$ python3 -m pip install semgrep
 
-# To try Semgrep without installation run via Docker
-$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
-```
-
-2.  Go to your app's root directory and run `semgrep scan --config auto`. This will scan your project with the default settings.
+For beginners, we recommend starting with the [Semgrep Cloud Platform](#option-2-getting-started-from-the-semgrep-cloud-platform-recommended) because it provides a visual interface, a demo project, result triaging and exploration workflows, and makes setup in CI/CD fast. Scans are still local and code isn't uploaded. Alternatively, you can also start with the CLI and navigate the terminal output to run one-off searches.
```

Comment:
Think using hyperlink anchors is an interesting idea, but here we have `#option-2-...` instead of the intended `#option-1-...`